osu.Game/OsuGame.cs

```diff
@@ -490,12 +480,13 @@ protected override void LoadComplete()
             dependencies.Cache(notifications);
             dependencies.Cache(dialogOverlay);
             dependencies.Cache(accountCreation);
+            dependencies.Cache(loginOverlay);
 
             chatOverlay.StateChanged += state => channelManager.HighPollRate.Value = state == Visibility.Visible;
 
             Add(externalLinkOpener = new ExternalLinkOpener());
 
-            var singleDisplaySideOverlays = new OverlayContainer[] { settings, notifications };
+            var singleDisplaySideOverlays = new OverlayContainer[] { settings, notifications, musicController, loginOverlay };
```

Comment:
music controller shouldn't be in here

Reply:
Is there a reason why?

Reply:
it's supposed to stay visible even when other overlays are opened/closed. at this point you've added every overlay inside `singleDisplay` which is defeating the purpose of it existing in the first place